client/app/components/AddressForm.jsx

```diff
@@ -7,20 +7,23 @@ import TextField from 'app/components/TextField';
 import SearchableDropdown from 'app/components/SearchableDropdown';
 import { createFilter } from 'react-select';
 
-export const AddressForm = ({ control, register, watch, isOrgPartyType }) => {
+export const AddressForm = ({ control, register, watch, isOrgPartyType, isIndividualPartyType, isHLROrSCForm }) => {
   const watchState = watch('state');
   const defaultState = useMemo(
     () => STATES.find((state) => state.label === watchState),
     [STATES, watchState]
   );
 
+  const optionalIndividualHLRField = Boolean(isIndividualPartyType && isHLROrSCForm);
```

Comment:
```suggestion
  const optionalIndividualClaimReviewField = Boolean(isIndividualPartyType && isHLROrSCForm);
```

Wondering since this variable is utilized both in HLR and SC intake flows if it'd be better to generalize the name.